formatter/fmt

```diff
@@ -47,9 +47,13 @@ npx prettier \
 echo 'Done formatting with prettier'
 
 echo 'Start formatting with eslint'
-npx eslint \
-  --fix \
-  {server/app/assets/javascripts,server/tailwind.config.js,test-support,browser-test}
+cd server
+npm install --silent
```

Comment:
Can we do also make sure to do these installs inside the dockerfile, so it's a one-time download (we would only need it here if the package.json is mounted at runtime and may change without a new docker build)

Reply:
It also runs `npm install` in the dockerfile. So here it will most likely reuse already downloaded files. Running npm intall here is needed because lint checks need to have dependencies present in order to do type-based checks. So when engineer updates `package.json` locally - we'll need to run `npm install` during formatting for eslint to properly account for the new/changed dependencies.